kubens

```diff
@@ -173,14 +175,19 @@ swap_namespace() {
 }
 
 main() {
-  if hash kubectl 2>/dev/null; then
+  set +u
```

Comment:
don't do this. the many `set -u`s you have below clearly shows it'll cause trouble someday when we forget to add `set -u` to a new code path. :)

instead use a one-off default like:

    "${KUBECTL:-}"

in the IF statement.

Reply:
🤦‍♂️... thanks